frontend-zs/src/components/projects/Home/ProjectCard.js

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import { Row, Card, Col } from 'react-bootstrap'
 
 
-function ProjectCard(project, index, isLeftSide, history) {
+function ProjectCard(project, isLeftSide, history) {
     const margin = isLeftSide ? 'ml-md-2' : 'mr-md-2'
 
     return (
-        <Row key={index}>
+        <Row key={project.id}>
```

Comment:
```suggestion
        <Row key={project._id}>
```
`project.id` is always undefined